api/utils/graph_context.py

```
@@ -1,92 +1,174 @@
+                sym_type, sym_type.capitalize()
             )
-            parts.append(f"**File:** `{sym.get('change_file')}` (lines {sym.get('start_line')}-{sym.get('end_line')})")
-            
-            # Add docstring if available
+
+            if args:
+                parts.append(f"### `{name}({args})` — {type_badge}")
+            else:
+                parts.append(f"### `{name}` — {type_badge}")
+
+            parts.append(f"**File:** `{file_path}` (lines {start}–{end})")
```

Ruff (0.15.0)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)